$NetBSD: patch-ac,v 1.1 2000/07/03 23:58:06 itohy Exp $

PS/LaTeX fix from the author of transfig.
This will be included in future releases.

--- fig2dev/dev/genpstex.c.orig	Wed Jan 12 10:15:26 2000
+++ fig2dev/dev/genpstex.c	Mon Jun 12 12:30:21 2000
@@ -136,7 +136,7 @@
 	gendev_null,
 	genpstex_t_text,
 	genlatex_end,
-	EXCLUDE_TEXT
+	INCLUDE_TEXT
 };
 
 struct driver dev_pstex = {
